backend/src/govsg/services/govsg-template.service.ts

```diff
@@ -0,0 +1,64 @@
+import { UploadData } from '@core/interfaces'
+import { UploadService } from '@core/services'
+import { CampaignGovsgTemplate } from '@govsg/models/campaign-govsg-template'
+import { GovsgMessage } from '@govsg/models/govsg-message'
+import { GovsgTemplate } from '@govsg/models/govsg-template'
+import { TemplateClient } from '@shared/templating'
+import { GovsgService } from '.'
+
+const templateCli = new TemplateClient({
+  xssOptions: { whiteList: { br: [] }, stripIgnoreTag: true },
+})
+
+export async function getFilledTemplate(
```

Comment:
don't really understand this. what is meant by `FilledTemplate`? also why return `null` instead of throwing an error

(note to self: to review more in-depth later)

Reply:
Getting the actual template with body and params from pivot table